### ISSUE TITLE
[not really an issue] AMD support? (+ context)

### ISSUE THREAD

**Issue by KevY007** [Nukem9/dlssg-to-fsr3#123]
I used a mod previously on Cyberpunk 2077 and RDR2. It basically replaced the game's implementation of DLSS with FSR 2.2 since neither RDR2 nor Cyberpunk 2077 had FSR 2.2 back then (or maybe still now).
It worked the same, just disable all other types of super sampling, game's FSR 2.1 implementation etc and just plainly enable DLSS (the mod allowed you to use DLSS options even if you own an AMD gpu) in game and it'll work.

So I was wondering if it's possible with this too?

Here are the relevant sources and links:
https://github.com/PotatoOfDoom/CyberFSR2
https://www.nexusmods.com/reddeadredemption2/mods/1640
https://www.nexusmods.com/cyberpunk2077/mods/3001

I apologize for raising an issue on a non issue but I was wondering if this is possible? I don't have much know-how regarding this topic to develop it myself but just knowing if it's possible or being pointed to the right direction is enough for me or someone else interested! Thank you.

**Comment by ForthtuN** [comment]
No, when using the DLSS option you are still on FSR and the way this mod in implemented it needs to be a RTX card.,

**Comment by amirjg** [comment]
> I used a mod previously on Cyberpunk 2077 and RDR2. It basically replaced the game's implementation of DLSS with FSR 2.2 since neither RDR2 nor Cyberpunk 2077 had FSR 2.2 back then (or maybe still now). It worked the same, just disable all other types of super sampling, game's FSR 2.1 implementation etc and just plainly enable DLSS (the mod allowed you to use DLSS options even if you own an AMD gpu) in game and it'll work.
> 
> So I was wondering if it's possible with this too?
> 
> Here are the relevant sources and links: https://github.com/PotatoOfDoom/CyberFSR2 https://www.nexusmods.com/reddeadredemption2/mods/1640 https://www.nexusmods.com/cyberpunk2077/mods/3001
> 
> I apologize for raising an issue on a non issue but I was wondering if this is possible? I don't have much know-how regarding this topic to develop it myself but just knowing if it's possible or being pointed to the right direction is enough for me or someone else interested! Thank you.

FSR 3 FG does work with AMD cards and there are already mods out there for it (paywalled though)
This specific mod Doesn't Because it uses Nvidia's Streamline...Nukem has built a workaround to use FSR3 FG instead of DLSS FG in games which have native DLSS FG support...So It is RTX only as stated everywhere on mod's pages

**Comment by hazre** [comment]
> FSR 3 FG does work with AMD cards and there are already mods out there for it (paywalled though) This specific mod Doesn't Because it uses Nvidia's Streamline...Nukem has built a workaround to use FSR3 FG instead of DLSS FG in games which have native DLSS FG support...So It is RTX only as stated everywhere on mod's pages

Which paywalled mods are you talking about, if you don't mind me asking?

**Comment by Aelnzon** [comment]
LukeFZ if im not rembering it wrong.
Altough the author already said the final version will be free on nexus.

**Comment by kedaitinh12** [comment]
For LukeFZ 0.7.2 with AMD support you can use here
https://drive.google.com/file/d/1-vgQPeIk_7wpgjiqqiqmoQUlrvDi0X6F/view?usp=drivesdk

And read here to know what file to use. Read redme in file download to know how to mod
https://docs.google.com/spreadsheets/d/1bGDjgkRahwNvD2XtqiHdZBRc6VB52UKsr1Qn1TmvSHo/edit#gid=0

**Comment by DRAGONTOS** [comment]
I don't know why he did that because it's illegal to put up a paywall, as this is licensed under GPLv3.

**Comment by kedaitinh12** [comment]
> I don't know why he did that because it's illegal to put up a paywall, as this is licensed under GPLv3.

LukeFZ 0.7.3
https://drive.google.com/file/d/19fZkma45w00Axu23IIiR-vppDEccN1VC/view?usp=sharing

**Comment by DJMalachite** [comment]
Any idea when Luke will make it free?
Tempted to subscribe to patreon as i tried those mods you uploaded and they work with a few bugs


> > I don't know why he did that because it's illegal to put up a paywall, as this is licensed under GPLv3.
> 
> LukeFZ 0.7.3 https://drive.google.com/file/d/19fZkma45w00Axu23IIiR-vppDEccN1VC/view?usp=sharing

**Comment by kedaitinh12** [comment]
Yes, i heard will free in feb 2024

**Comment by DJMalachite** [comment]
> Yes, i heard will free in feb 2024

Oh sweet.
